docs/learn/learn-assets.md

```diff
@@ -108,6 +108,32 @@ that will fail if execution kills the sending account.
 transfers. For example, if an asset has a minimum balance of 10 and an account has a balance of 25,
 then an attempt to transfer 20 units would transfer all 25.
 
+:::info
+
+Before transferring an asset make sure the receiver account has enough funds to cover for the
+existential deposit and transaction fees for future transfers. Failing to do so will cause the asset
+transfer to fail.
```

Comment:
If the transferred asset is a sufficient asset and the amount sent is more than the ED requirement for that asset, it should be okay. Also, asset transfer would not fail in that case. The user will just not be able to move those assets.